docs/source/concept_models/vivarium_swissre_cervicalcancer/cervical_cancer_concept_model.rst

```diff
@@ -537,18 +534,18 @@ Treatment for benign cervical cancer
      - Wang et al. 2015
      - It's an arbitrary number greater than 20.7%.
    * - Target cervical cancer screening coverage in 2030
-     - 70%
-     - WHO cervical cancer elimination strategy
+     - 50%
      - 
+     - by assumption
    * - Initial HPV vaccination coverage
      - 10%
      - 
      - The current HPV vaccination rates remain low in China, no data has
        identified.
    * - Target HPV vaccination coverage in 2030
-     - 90%
-     - WHO cervical cancer elimination strategy
+     - 40%
      - 
+     - by assumption
```

Comment:
Flagging to look into literature again to guide these coverage numbers, I have it noted in Asana to keep track.